ecommerce/basket/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+# Create your models here.
+from core.models import BaseAbstractModel
+from customers.models import Customer
+from django.utils.translation import gettext_lazy as _
+
+from products.models import Product
+
+
+class Status(models.TextChoices):
+    OPEN = "open", _("Open")
+    SUBMITTED = "submitted", _("Submitted")
+    MERGED = "merged", _("Merged")
+
+
+class Basket(BaseAbstractModel):
+    # when customer is deleted the basket can be remain
+    customer = models.ForeignKey(Customer, null=True, on_delete=models.SET_NULL)
```

Comment:
blank=True parametresini de eklersek, formlardan blank olarak gelmesine de izin vermis oluruz. basket'i olan bir customer'in silinmesine izin vermek guzel bir fikir olmayabilir.